src/test/resources/pages/InstructorCourseDetailsEmptyCourse.html

```diff
@@ -335,7 +332,7 @@
                 </div>
                 
                 <p class="form-group text-muted">
-                    The default visibility for your comment is private. You may change it using the ‘show visibility options’ button above.
+                    The default visibility for your comment is private. You may change it using the 'show visibility options' button above.
```

Comment:
This is from `commentArea.tag`.
We should probably retain the curved quotation marks.